### PR TITLE
Allow switching the EE jit on or off through the settings window.

### PR DESCRIPTION
**Pull request by Ziemas**
A copy and paste job from the VU1 setting implementation.

### CODE REVIEW

File: src/core/emulator.cpp [hunk 1]
```diff
@@ -319,14 +319,28 @@ void Emulator::set_skip_BIOS_hack(SKIP_HACK type)
     skip_BIOS_hack = type;
 }
 
-void Emulator::set_vu1_mode(VU_MODE mode)
+void Emulator::set_ee_mode(CPU_MODE mode)
+{
+  switch (mode)
+    {
```
Comment (PSI-Rockin) [inline src/core/emulator.cpp]: Indent these curly brackets on the same column as the switch statement.

File: src/qt/emuwindow.cpp [hunk 2]
```diff
@@ -665,18 +666,34 @@ void EmuWindow::show_render_view()
     stack_widget->setCurrentIndex(1);
 }
 
+void EmuWindow::set_ee_mode()
+{
+  CPU_MODE mode;
+  if (Settings::instance().ee_jit_enabled)
+    {
```
Comment (PSI-Rockin) [inline src/qt/emuwindow.cpp:673]: Indent these curly brackets properly.

File: src/qt/emuthread.cpp [hunk 3]
```diff
@@ -34,7 +34,14 @@ void EmuThread::set_skip_BIOS_hack(SKIP_HACK skip)
     load_mutex.unlock();
 }
 
-void EmuThread::set_vu1_mode(VU_MODE mode)
+void EmuThread::set_ee_mode(CPU_MODE mode)
+{
+  load_mutex.lock();
```
Comment (PSI-Rockin) [inline src/qt/emuthread.cpp]: Use four spaces for indentation, not two.

**Review by PSI-Rockin — CHANGES_REQUESTED** [review]

**Comment by Ziemas** [comment]
Fixed formatting and squashed fixups.

**Pull request merged**